Kingfisher/UIImageView+Kingfisher.swift

```diff
@@ -110,6 +111,21 @@ public extension UIImageView {
                          progressBlock: DownloadProgressBlock?,
                      completionHandler: CompletionHandler?) -> RetrieveImageTask
     {
+        var _indicator = UIActivityIndicatorView(activityIndicatorStyle:.Gray)
```

Comment:
@vfn Sure. I guess I will improve this p-r later since there is no response from @alexliubj